lib/commands/general.js

```diff
@@ -20,4 +22,27 @@ commands.getPageSource = async function () {
   }
 };
 
+commands.getWindowSize = async function () {
+
+  var commandObject = {
+    name: `getWindowSize`
+  };
+  var commandJSON = JSON.stringify(commandObject);
+
+  let data = await this.executeSocketCommand(commandJSON);
+
+  let result;
+  try {
+    result = JSON.parse(data);
+  } catch (e) {
+    throw new Error("Bad response from window_size");
+  }
```

Comment:
Is this what will be thrown if someone uses this version of the appium driver (js) with an older version of the engine? (i.e. one that doesn't support this command?)